app/assets/scripts/views/field-report-form/index.js

```diff
@@ -748,8 +748,9 @@ class FieldReportForm extends React.Component {
                 {this.renderErrorSummary()}
 
                 <div className='form__actions'>
-                  <button type='button' className={c('button button--secondary-bounded', {disabled: this.state.step <= 1})} title='Go back to previous step' onClick={this.onStepBackClick}>Back</button>
+                  <button type='button' className={c('button button--base-plain', {disabled: this.state.step <= 1})} title='Go back to previous step' onClick={this.onStepBackClick}>Back</button>
                   <button type='submit' className='button button--secondary-filled' title='Save and continue'>Save and continue</button>
+                  <button type='submit' className='button button--secondary-bounded' title='Save and continue'>Save as Draft</button>
```

Comment:
@ascalamogna I think we should avoid having two `type=submit` elements, as that could cause weirdness or unexpected behavior when you hit enter in the form. Also I think the title of the draft button needs to be changed.